packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -100,7 +99,7 @@ export const DataTable = <T extends DomainObject>({
                 onClick={onRowClick}
                 rowData={row}
                 rowKey={String(idx)}
-                noLines={noLines}
+                dense={dense}
```

Comment:
removed the lines - but also reduced font and padding, to make the dense version denser. I think it was a bit too much with all the lines